mne/io/curry/curry.py

```diff
@@ -149,10 +158,34 @@ def _read_curry_parameters(fname):
                     unit_dict[type] = data_unit.replace(" ", "") \
                         .replace("\n", "").split("=")[-1]
 
+    # look for CHAN_IN_FILE sections, which may or may not exist; issue #8391
+    types = ["meg", "eeg", "misc"]
+    chanidx_in_file = _read_curry_lines(fname,
+                                        ["CHAN_IN_FILE" +
+                                         CHANTYPES[key] for key in types])
+
     n_samples = int(param_dict["numsamples"])
     sfreq = float(param_dict["samplefreqhz"])
     time_step = float(param_dict["sampletimeusec"]) * 1e-6
     is_ascii = param_dict["dataformat"] == "ASCII"
+    n_channels = int(param_dict["numchannels"])
+    try:
+        dt_start = datetime(int(param_dict["startyear"]),
+                            int(param_dict["startmonth"]),
+                            int(param_dict["startday"]),
+                            int(param_dict["starthour"]),
+                            int(param_dict["startmin"]),
+                            int(param_dict["startsec"]),
+                            int(param_dict["startmillisec"]) * 1000,
+                            datetime.now().astimezone().tzinfo)\
```

Comment:
This line implies that someone reading the file in Paris will think the recording started at a different absolute/UTC time than someone reading the file in New York. This seems like it can't be correct. Hopefully the file format either stores in UTC, or says what time zone it's stored in? If not, then I think we should just set it to UTC and know that it's wrong if the data were recorded in a different time zone. That way at least anyone who reads the file from any physical location will get the same wrong time zone, instead of all getting potentially different ones (and it only being correct in the time zone where the data were collected).

Reply:
Fair enough.... I thought about that, and I guess it is the better approach.  I will change it.
From what I can see, the times in the file (StartHour) must be local times.  I have 16 recordings I'm looking at, and the time don't make sense if you interpret them as UTC.
That being said, I wish I had access to an official description of the Curry file formats.   Everything I am doing is based on my best guesses about what I see in my example files.

Reply:
Some files have a `StartOffsetUTCMin`, perhaps this could be used?

Reply:
(I assume this is the UTC offset in minutes, so you would just add this to the `startmin` above and set the `tzinfo` to be `UTC`)

Reply:
... actually in some of our test files this value is 9999 even though the other time values seem reasonable, so now I'm not so sure. But maybe this was hacked/changed? I would check with your own recorded files locally to see if this parameter is reasonable, and if it is and it gives you the right recording onset time, assume our test files are weird/broken.

Reply:
Strange.  I don't have `StartOffsetUTCMin`  in any of my example files.  And 9999 certainly seems like a strange value.
I noticed some other strange values in some of the test files, such as `StartSec             =  11280` in file **test_bdf_stim_channel Curry 7.dap** (which was also blowing up my datetime construction until I added the try/except).   So I question whether all the test files are exactly as they would be coming from the real Curry software.

Reply:
@DiGyt do you remember where the Curry files came from? They have a weird `StartOffsetUTCMin=9999` value in them.